staking-pool/README.md

```diff
@@ -87,6 +87,11 @@ Calls the internal function to distribute rewards if the blockchain epoch switch
 Before every action the contract calls method `internal_ping`.
 This method distributes rewards towards active delegators when the blockchain epoch switches.
 The rewards might be given due to staking and also because the contract earns gas fee rebates for every function call.
+Note, the if someone accidentally (or intentionally) transfers tokens to the contract (without function call), then
+tokens form the transfer will be distributed to the active stake participants of the contract in the next epoch.
+Note, in a rare scenario, where the owner withdraws tokens and while the call is not processed deletes their account, the
```

Comment:
"while the call is not processed" shouldn't this be "while the call is being processed"?